gtnet/data/.ipynb_checkpoints/test_model-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Remove

Reply:
removed checkpoints, not sure why .gitignore in the parent folder can't keep these out?